CRM/Core/Resources.php

```diff
@@ -818,6 +818,19 @@ public function coreResourceList($region) {
     // Allow hooks to modify this list
     CRM_Utils_Hook::coreResourceList($items, $region);
 
+    // Oof, existing listeners would expect $items to typically begin with 'bower_components/' or 'packages/'
+    // (using an implicit base of `[civicrm.root]`). We preserve the hook contract and cleanup $items post-hook.
```

Comment:
This is the most complex part of this PR but my  take on  it is it would fail  very broadly if it  were going to. I'm pretty sure the new mailing screen would not  load if this were broken (it does load)

Reply:
Quite right - if these things didn't work, then everything would be wrong in... pretty much every screen. It'd be missing `jQuery`, etc. A problem would be pretty obvious.